app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+class FileNotFound(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    operation, file_name, destination = command.split()
+
+    if operation != "cp" or file_name == destination:
+        print("File error")
```

Comment:
> It must do nothing in case the user is trying to copy file to file with the same name 


but try not to use blank return here, make a backward condition